composer.json

```diff
@@ -117,6 +117,7 @@
         }
     },
     "replace": {
+        "zendframework/zendframework1": ">=1.12.20",
```

Comment:
why `>=1.12.20`? Maybe we should replace only versions greater than `>1.12.20` 
https://github.com/zendframework/zf1/tags

Reply:
it's fine to cover last officially released version. zf1-future and diablomedia/zf1 have both configured it that way too.